lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -469,16 +470,41 @@ isSubrangeOf r1 r2 =
                                   Stake Pools
 -------------------------------------------------------------------------------}
 
--- | Represent stake pool identifier. Note that the internal representation is
--- left open currently, until we figure out a better type for those.
-newtype PoolId = PoolId
-    { getPoolId :: Text }
+-- | Represent stake pool identifier.
+newtype PoolId = PoolVRFPubKey { getPoolId :: ByteString }
```

Comment:
Maybe
```suggestion
newtype PoolId = PoolVRFPubKey { getVRFPubKey :: ByteString }
```
_not sure if I've caused havoc by suggesting to call the type PoolId, but the constructor PoolVRFPubKey_

Reply:
I am fond of `newtype PoolId = PoolId { getPoolId :: ByteString }` but that is more my preference to stick to convention we use throughout the code ;-)